### ISSUE TITLE
A small script for managing maps

### ISSUE THREAD

**Issue by farooqkz** [MT-CTF/maps#306]
I am thinking about a small program for map management(enabling/disabling per mode or entirely) and printing various information about them. What language do you think is the best choice for the job? my own vote is Python

**Comment by LoneWolfHT** [comment]
I was planning something like this as a part of the `ctf_map` mod, enabled when in map editing mode

**Comment by farooqkz** [comment]
> I was planning something like this as a part of the `ctf_map` mod, enabled when in map editing mode.
> 
> That way it comes with CTF by default, and all you need to do to use it is add it to the trusted mods to get write permissions to the mod folders (IIRC)

hmm but then you'll have to launch minetest. right? It won't be convenient IMO.

**Comment by LoneWolfHT** [comment]
non-techy map makers will have to install python, and launch a terminal to run it

**Comment by farooqkz** [comment]
> non-techy map makers will have to install python, and launch a terminal to run it

What about Rust then? It could have a TUI or a GUI.

**Comment by LoneWolfHT** [comment]
Doing it in Minetest will make it much easier to maintain/contribute to, and I want it to be a part of the map editor

**Comment by farooqkz** [comment]
> Doing it in Minetest will make it much easier to maintain/contribute to, and I want it to be a part of the map editor

I see your point. But running Minetest just to do map management doesn't seem the right solution for this. Perhaps only for enable/disable. But not for the entire analysis of the maps.

**Comment by LoneWolfHT** [comment]
Maybe do an unofficial tool then
